index.md

```diff
@@ -115,22 +115,18 @@ title: Covid Shots
 <select name="states" multiple="" class="ui fluid dropdown" id="states">
 </select>
 
-<button class="ui primary button" id="getDistrictsBtn">
-  Get districts
-</button>
 
+<br>
 <h5 class="ui header">Districts</h5>
 <select name="districts" multiple="" class="ui fluid dropdown disabled" id="districts">
     <option>Select districts</option>
 </select>
 
-
+<br>
```

Comment:
Remove this<br />. Vertical space between the two dropdowns without this seems enough. One reason to remove getdistrict and centre buttons was to reduce vertical space so that the site looks better on mobile